File: generators/web/prototype/org.obeonetwork.informationsystem.gen.web.proto/asset/js/app/model/Proxy.js
/*******************************************************************************
 * Copyright (c) 2012 Obeo.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 * 
 * Contributors:
 *     Obeo - initial API and implementation
 *******************************************************************************/

/*
 * @autor hugo.marchadour@obeo.fr
 * @version 0.9
 * @since 0.8
 * define the ember proxy model
 * @see http://emberjs.com/#toc_the-ember-object-model
 */
define(["ember"], function(/*Em*/) {

	var proxy = {};

	/*
	 * The class definition
	 * @param {long} id
	 * @param {String} description

	*/
	proxy.Class = Em.Object.extend({
		id : null, 
		desc : null
	});

	return proxy;

});

File: samples/plugins/org.obeonetwork.informationsystem.gen.web.sample/genWithPlay/app/models/Proxy.java
/*******************************************************************************
 * Copyright (c) 2017 Obeo.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 *
 * Contributors:
 *     Obeo - initial API and implementation
 *******************************************************************************/
package models;


public class Proxy {

	private Long id;
	
	private String desc;
	

	
	public Proxy(Long id, String desc) {

		super();
		
		this.id = id;
		this.desc = desc;
	}


	public Long getId() {
		return id;
	}


	public void setId(Long id) {
		this.id = id;
	}


	public String getDesc() {
		return desc;
	}


	public void setDesc(String desc) {
		this.desc = desc;
	}

}
